lib/go/test/topshot_test.go

```diff
@@ -464,7 +464,7 @@ func TestMintNFTs(t *testing.T) {
 		assert.Equal(t, cadence.NewBool(true), result)
 
 		result = executeScriptAndCheck(t, b, templates.GenerateGetCollectionIDsScript(env), [][]byte{jsoncdc.MustEncode(cadence.Address(topshotAddr))})
-		idsArray := cadence.NewArray([]cadence.Value{cadence.NewUInt64(6), cadence.NewUInt64(5), cadence.NewUInt64(1), cadence.NewUInt64(4), cadence.NewUInt64(3), cadence.NewUInt64(2)})
+		idsArray := cadence.NewArray([]cadence.Value{cadence.NewUInt64(2), cadence.NewUInt64(1), cadence.NewUInt64(5), cadence.NewUInt64(4), cadence.NewUInt64(3), cadence.NewUInt64(6)})
```

Comment:
Changed the ordering of the array here, shouldn't be ok I think - it still contains the same elements

Reply:
yep, the ordering changes a lot. Not sure why

Reply:
It wasn't on the previous version 😓